### ISSUE TITLE
Wrong version of spring activated when executing commands with spring

### ISSUE THREAD

**Issue by caleb** [asok/projectile-rails#50]
Firstly, let me say that this plugin is really helpful. It's way more featureful than I expected going in (I was just looking for a way to jump around my project).

I have a problem that when I run the commands that use spring (console, rake, etc), I get an error:

```
You have already activated spring 1.2.0, but your Gemfile requires spring 1.1.3. Prepending `bundle exec` to your command may solve this.
```

The project I am working on is Rails 4.1 but on this version of ruby I have Rails 4.2 installed as well. I think the problem is that when projectile-rails runs commands with spring it uses plain old `spring` which uses the newest version of spring.

I think a solution would be to either run with `bundle exec` as specified in the error message or to run `bin/spring` from the project directory since that is a wrapper that runs the version of spring in the Gemfile.

What do you think? I could take a crack at a patch, but while my lisp-fu is alright, debugging and hacking emacs is not something I have much experience with.

Thanks again for this project. I am already more productive with it.

**Comment by asok** [comment]
Hey, cool you like it :)
Yes I think your suggestion is alright. Make a PR if you wish to.

**Comment by asok** [comment]
Closed by #51
